Orange/widgets/data/owrank.py

```diff
@@ -523,9 +523,6 @@ def commit(self):
         selected_attrs = []
         if self.data is not None:
             attributes = self.data.domain.attributes
-            if len(attributes) == len(self.selected_rows):
```

Comment:
You can delete the row above as well and use the long version below (it is now only used once)